src/modules/75-cf/pages/FFUIApp/FFUIApp.tsx

```diff
@@ -64,7 +64,7 @@ const FFUIApp: FC = () => (
     customComponents={{
       ContainerSpinner,
       Description,
-      EvaluationModal,
+      PolicyManagementEvaluationModal,
```

Comment:
This will be a breaking change to this MFE, better to alias it and do the same for the other MFEs and MFE typings
```suggestion
      EvaluationModal: PolicyManagementEvaluationModal,
```